Gruntfile.js

```diff
@@ -735,34 +728,50 @@ module.exports = function (grunt) {
 
   grunt.loadNpmTasks('grunt-angular-templates');
 
+  // ex:
+  // grunt test-unit
+  // grunt test-unit --skipRebuild (to save time between test runs)
   // karma must run prior to coverage since karma will generate the coverage results
-  grunt.registerTask('test-unit', [
-    'clean:server',
-    'concurrent:test',
-    'postcss',
-    'connect:test',
-    'karma'
-    // 'coverage' - add back if we want to enforce coverage percentages
-  ]);
+  grunt.registerTask('test-unit',
+    grunt.option('skipRebuild') ?
+      [
+        'connect:test',
+        'karma'
+      ] :
+      [
+        'clean:server',
+        'concurrent:test',
+        'postcss',
+        'connect:test',
+        'karma'
+        // 'coverage' - add back if we want to enforce coverage percentages
+      ]
+    );
 
-  // test as an alias to unit.  after updating protractor,
-  // will make test an alias for both unit & e2e
+  // alias
+  // TODO: 'test' should run both unit & e2e tests
   grunt.registerTask('test', ['test-unit']);
 
+  // ex:
+  // grunt test-integration
+  // grunt test-integration --baseUrl=      (for build environments)
+  // grunt test-integration --skipRebuild   (to save time between test runs)
   grunt.registerTask('test-integration',
-    // if a baseUrl is defined assume we dont want to run the local grunt server
-    grunt.option('baseUrl') ?
-      [isMac ? 'protractor:mac' : 'protractor:default'] :
+    (grunt.option('baseUrl') || grunt.option('skipRebuild') ) ?
+      ['protractor:default'] :
       [
         'clean:server',
         'development-build',
         'postcss',
         'connect:test',
         'add-redirect-uri',
-        (isMac ? 'protractor:mac' : 'protractor:default'),
+        'protractor:default',
         'clean:server'
       ]
-  );
+    );
+
+  // alias
+  grunt.registerTask('test-e2e', ['test-integration']);
```

Comment:
There's actually a specific e2e suite that is just part of the integration tests. I'd remove this alias since they're not exactly the same.